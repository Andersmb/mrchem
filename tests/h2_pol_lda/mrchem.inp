# vim:syntax=sh:

world_prec = 1.0e-3               # Overall relative precision
world_size = 6                    # Size of simulation box 2^n

MPI {
  numerically_exact = true        # Guarantee identical results in MPI
}

Molecule {
  angstrom = true
$coords
H      0.0000 0.0000   -0.3705
H      0.0000 0.0000    0.3705
$end
}

WaveFunction {
  method = DFT                    # Wave function method (HF or DFT)
  restricted = false              # Unrestricted calculation
}

DFT {
$functionals
LDA
$end
}

Properties {
  scf_energy = true               # Compute ground state energy
  polarizability = true           # Compute electric polarizability
}

Polarizability {
  frequency = [0.0]               # Static polarizability 
}

SCF {
  run = false                     # Use initial guess, don't optimize
  initial_guess = GTO             # Reading precomputed GTO MOs
}

Response {
  kain = 3                        # Length of KAIN iterative history
  max_iter = 10
  orbital_thrs = 1.0e-2           # Convergence threshold in orbital residual
  directions = [0,0,1]            # Compute only in z direction
}

