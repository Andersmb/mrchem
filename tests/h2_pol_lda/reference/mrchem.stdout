


************************************************************
***     __  __ ____   ____ _                             ***
***    |  \/  |  _ \ / ___| |__   ___ _ __ ___           ***
***    | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \          ***
***    | |  | |  _ <| |___| | | |  __/ | | | | |         ***
***    |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|         ***
***                                                      ***
***    VERSION 0.0.1 (rev. 996f83d5)                      ***
***                                                      ***
***    Stig Rune Jensen <stig.r.jensen@uit.no>           ***
***    Luca Frediani    <luca.frediani@uit.no>           ***
***    Peter Wind       <peter.wind@uit.no>              ***
***                                                      ***
************************************************************


+++ Serial execution


------------------------------------------------------------

 MRCPP version   : X.Y.Z
 Git revision    : e5c44e2

 Linear algebra  : EIGEN
 Parallelization : NONE

------------------------------------------------------------




============================================================
                  MultiResolution Analysis                  
------------------------------------------------------------
 polynomial order = 5
 polynomial type  = Interpolating
------------------------------------------------------------
 unit length      = 32.000000
 total boxes      = 8
 boxes            = [           2           2           2 ]
 lower bounds     = [  -32.000000  -32.000000  -32.000000 ]
 upper bounds     = [   32.000000   32.000000   32.000000 ]
 total length     = [   64.000000   64.000000   64.000000 ]
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     H           0.00000e+00   0.00000e+00  -7.00143e-01
  2     H           0.00000e+00   0.00000e+00   7.00143e-01
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   0.00000e+00
============================================================


============================================================
                Setting up nuclear potential
------------------------------------------------------------
 Nr  Element         Charge        Precision     Smoothing 
------------------------------------------------------------
  1     H            1.00000e+00   1.00000e-03   1.63241e-02
  2     H            1.00000e+00   1.00000e-03   1.63241e-02
------------------------------------------------------------
 Projecting potential                  776      2.45284e-01
============================================================


============================================================
                Setting up Gaussian-type MOs
------------------------------------------------------------
    n  Spin  Occ                           SquareNorm
------------------------------------------------------------
    0    a    1                       9.999999974573798e-01
------------------------------------------------------------
                 Wall time: 1.77353e-01 sec 
============================================================


============================================================
                Setting up Gaussian-type MOs
------------------------------------------------------------
    n  Spin  Occ                           SquareNorm
------------------------------------------------------------
    0    b    1                       9.999999974573796e-01
------------------------------------------------------------
                 Wall time: 1.76128e-01 sec 
============================================================


============================================================
 OrbitalVector:   2 orbitals     2 occupied     2 electrons
------------------------------------------------------------
   n  RankID           Norm          Spin Occ      Error    
------------------------------------------------------------
   0     0       9.999999974574e-01    a   1    1.00000e+00
   1     0       9.999999974574e-01    b   1    1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Coulomb density                       392      1.73102e-01
 Coulomb potential                       8      2.49541e-01
 XC alpha density                      392      8.96390e-02
 XC beta density                       392      8.85401e-02
 XC evaluate xcfun                    1176      5.82220e-01
 XC energy                             392      2.87060e-05
 XC potential                          784      1.30939e-02
 XC grid size                          392           594432
 XC grid change                       -384          -580608
------------------------------------------------------------
                 Wall time: 1.19859e+00 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         2.550113060000000e-01
 Potential part                       5.979056420000000e-01
------------------------------------------------------------
                 Wall time: 8.53108e-01 sec 
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        6.00298e-04
 Diagonalizing matrix                           1.21300e-05
 Rotating orbitals                              2.23105e-02
------------------------------------------------------------
                 Wall time: 2.29865e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Coulomb density                       392      1.69075e-01
 Coulomb potential                       8      2.47953e-01
 XC alpha density                      392      8.83010e-02
 XC beta density                       392      8.75628e-02
 XC evaluate xcfun                    1176      5.82780e-01
 XC energy                             392      1.97740e-05
 XC potential                          784      1.31844e-02
 XC grid size                          392           594432
 XC grid change                       -384          -580608
------------------------------------------------------------
                 Wall time: 1.19123e+00 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 6.07238e-01 sec 
============================================================


 XC alpha density                      392      9.11404e-02
 XC beta density                       392      8.79388e-02
 XC evaluate xcfun                    2352      1.37043e+00
 XC potential                         1960      3.33095e-02
 XC grid size                          392           594432
 XC grid change                       -384          -580608
============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Coulomb density                       392      1.61685e-01
 Coulomb potential                       8      2.47398e-01
 XC alpha density                      392      8.58253e-02
 XC beta density                       392      8.59165e-02
 XC evaluate xcfun                    1176      5.82404e-01
 XC energy                             392      2.07980e-05
 XC potential                          784      1.23360e-02
 XC grid size                          392           594432
 XC grid change                       -384          -580608
------------------------------------------------------------
                 Wall time: 1.17805e+00 sec 
============================================================




####################### SCF cycle  1 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              1.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Perturbed Coulomb density               8      7.58680e-05
 Coulomb potential                       8      2.89013e-03
 XC perturbed alpha density              8      5.40720e-05
 XC perturbed beta density               8      4.98430e-05
------------------------------------------------------------
                 Wall time: 4.25322e-03 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
             F_0 phi_1                          7.49200e-06
 (1 - rho_0) V_1 phi_0                          9.80782e-01
------------------------------------------------------------
                 Wall time: 9.87906e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.45809e+00   264    0.00000e+00     0   3.86088e+00
  1    1.45809e+00   264    0.00000e+00     0   3.88892e+00
------------------------------------------------------------
                 Wall time: 7.75104e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.91280e-05
 Copy orbitals                                  1.87600e-06
 Copy orbital updates                           7.33080e-03
------------------------------------------------------------
                 Wall time: 7.40431e-03 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0   1.45809e+00   1.45809e+00        328    a    1    0
  1   1.45809e+00   1.45809e+00        328    b    1    0
------------------------------------------------------------
 Total error:                            2.06205e+00    0
============================================================


============================================================
                    Value                  Update      Done 
------------------------------------------------------------
 Property     -5.524041906252971e+00    -5.52404e+00    1
============================================================


################ Wall time: 9.15526e+00 sec ################




####################### SCF cycle  2 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              1.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Perturbed Coulomb density             328      2.08682e-01
 Coulomb potential                     200      1.40675e+00
 XC perturbed alpha density            328      1.07268e-01
 XC perturbed beta density             328      1.09730e-01
------------------------------------------------------------
                 Wall time: 1.83423e+00 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
             F_0 phi_1                          7.38200e-06
 (1 - rho_0) V_1 phi_0                          9.87601e-01
------------------------------------------------------------
                 Wall time: 9.97809e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.77198e+00   392    0.00000e+00     0   7.55211e+00
  1    1.77198e+00   392    0.00000e+00     0   7.54637e+00
------------------------------------------------------------
                 Wall time: 1.51006e+01 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             5.72600e-06
 Setup linear system                            2.20792e-02
 Solve linear system                            2.05110e-05
 Expand solution                                2.55484e-02
------------------------------------------------------------
                 Wall time: 4.77927e-02 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0   1.85803e+00   3.17152e-01        328    a    1    0
  1   1.85803e+00   3.17152e-01        328    b    1    0
------------------------------------------------------------
 Total error:                            4.48521e-01    0
============================================================


============================================================
                    Value                  Update      Done 
------------------------------------------------------------
 Property     -7.125003681733141e+00    -1.60096e+00    1
============================================================


################ Wall time: 1.83911e+01 sec ################




####################### SCF cycle  3 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              1.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Perturbed Coulomb density             392      2.54136e-01
 Coulomb potential                     200      1.49918e+00
 XC perturbed alpha density            392      1.29444e-01
 XC perturbed beta density             392      1.29678e-01
------------------------------------------------------------
                 Wall time: 2.01442e+00 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
             F_0 phi_1                          7.70000e-06
 (1 - rho_0) V_1 phi_0                          9.67527e-01
------------------------------------------------------------
                 Wall time: 9.74601e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.86508e+00   392    0.00000e+00     0   7.68502e+00
  1    1.86508e+00   392    0.00000e+00     0   7.67497e+00
------------------------------------------------------------
                 Wall time: 1.53622e+01 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.35664e-03
 Setup linear system                            6.03948e-02
 Solve linear system                            2.22150e-05
 Expand solution                                4.27606e-02
------------------------------------------------------------
                 Wall time: 1.04753e-01 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0   1.87077e+00   2.22539e-02        456    a    1    0
  1   1.87077e+00   2.22539e-02        456    b    1    0
------------------------------------------------------------
 Total error:                            3.14718e-02    0
============================================================


============================================================
                    Value                  Update      Done 
------------------------------------------------------------
 Property     -7.211854409024744e+00    -8.68507e-02    1
============================================================


################ Wall time: 1.89375e+01 sec ################




####################### SCF cycle  4 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              1.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Perturbed Coulomb density             392      2.45761e-01
 Coulomb potential                     200      1.52961e+00
 XC perturbed alpha density            392      1.32541e-01
 XC perturbed beta density             392      1.30695e-01
------------------------------------------------------------
                 Wall time: 2.04061e+00 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
             F_0 phi_1                          7.16000e-06
 (1 - rho_0) V_1 phi_0                          1.01906e+00
------------------------------------------------------------
                 Wall time: 1.02778e+00 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.87097e+00   392    0.00000e+00     0   7.82701e+00
  1    1.87097e+00   392    0.00000e+00     0   7.86138e+00
------------------------------------------------------------
                 Wall time: 1.56906e+01 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             3.16238e-03
 Setup linear system                            8.81133e-02
 Solve linear system                            9.80100e-06
 Expand solution                                4.85636e-02
------------------------------------------------------------
                 Wall time: 1.40074e-01 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0   1.87176e+00   3.95465e-03        456    a    1    1
  1   1.87176e+00   3.95465e-03        456    b    1    1
------------------------------------------------------------
 Total error:                            5.59273e-03    1
============================================================


============================================================
                    Value                  Update      Done 
------------------------------------------------------------
 Property     -7.222092470924626e+00    -1.02381e-02    1
============================================================


################ Wall time: 1.93844e+01 sec ################


============================================================
                      Convergence rate
------------------------------------------------------------
Iter    OrbError       Property                   Update  
------------------------------------------------------------
  1    2.06205e+00    -5.524041906252971e+00   -5.52404e+00
  2    4.48521e-01    -7.125003681733141e+00   -1.60096e+00
  3    3.14718e-02    -7.211854409024744e+00   -8.68507e-02
  4    5.59273e-03    -7.222092470924626e+00   -1.02381e-02
------------------------------------------------------------
                      SCF converged!!!                      
============================================================


============================================================
                 Calculating polarizability
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 1.34660e+00 sec 
============================================================


============================================================
                        Fock matrix
------------------------------------------------------------
-3.45284e-01  0.00000e+00
 0.00000e+00 -3.45284e-01
============================================================


============================================================
                      Orbital energies
------------------------------------------------------------
    n  spin  occ                            epsilon  
------------------------------------------------------------
    0    a    1                      -3.452838805081183e-01
    1    b    1                      -3.452838805081182e-01
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     H           0.00000e+00   0.00000e+00  -7.00143e-01
  2     H           0.00000e+00   0.00000e+00   7.00143e-01
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   0.00000e+00
============================================================


                                                            
============================================================
                         SCF Energy                         
------------------------------------------------------------
                                                            
 Sum orbital energy:                 -6.905677610162365e-01
 Kinetic energy:                      1.073485975656848e+00
 E-N energy:                         -3.566631920969574e+00
 Coulomb energy:                      1.328364211782457e+00
 Exchange energy:                     0.000000000000000e+00
 X-C energy:                         -6.538705241579309e-01
 El. external field energy:           0.000000000000000e+00
 Nuc. external field energy:          0.000000000000000e+00
                                                            
------------------------------------------------------------
                                                            
 Electronic energy:                  -1.818652257688199e+00
 Nuclear energy:                      7.141393372477786e-01
                                                            
------------------------------------------------------------
                                                            
 Total energy       (au)             -1.104512920440421e+00
                    (kJ/mol)         -2.899898252901416e+03
                    (kcal/mol)       -6.930923162092078e+02
                    (eV)             -3.005532637080431e+01
                                                            
============================================================
                                                            

                                                            
============================================================
                   Polarizability tensor                    
------------------------------------------------------------
                                                            
 Frequency:        (au)                    0.0000000000e+00
                                                            
------------------------------------------------------------
                                                            
 Isotropic average (au)                    2.4073641570e+00
 Isotropic average (SI)      TO BE FIXED                    
                                                            
-------------------------- Tensor --------------------------
                                                            
 [ 0.0000000000e+00,  0.0000000000e+00,  0.0000000000e+00] 
 [ 0.0000000000e+00,  0.0000000000e+00,  0.0000000000e+00] 
 [ 1.4977343513e-15, -3.2730815869e-15,  7.2220924709e+00] 
                                                            
============================================================
                                                            



************************************************************
***                                                      ***
***                    Exiting MRChem                    ***
***                                                      ***
***               Wall time: 7.456604e+01                ***
***                                                      ***
************************************************************


